DXRModules/DeusEx/Classes/DXRFixup.uc

```diff
@@ -207,13 +207,9 @@ function PreFirstEntry()
     FixBeamLaserTriggers();
     SpawnDatacubes();
     AntiEpilepsy();
+
+    foreach AllActors(class'#var(prefix)Lamp', lmp) lmp.InitLight();
```

Comment:
we don't currently injects for other mods, so the InitLight function won't exist